docs/userman/advanced_features/auth/authentication.md

```diff
@@ -427,3 +427,73 @@ connect to the directory service:
         # Returns a valid Credentials instance if the LDAP server validates the password
         # "pass123" for user "user1". Raises an InvalidCredentials exception otherwise.
         ```
+
+# Microsoft Entra ID Authenticator
+
+Taipy also provides support for Microsoft Entra ID authentication.
+
+An authenticator using the *Entra ID* protocol has two specific parameters that need to be provided
+in order to properly connect to the Microsoft Entra ID service:
+
+- *client_id*: The client ID of the Entra ID application. The application must be registered in the
+    Azure Entra ID portal and have the required permissions including the "User.Read" and
+    "GroupMember.Read.All" permissions.
+- *tenant_id*: The tenant ID of the Entra ID organization.
+
+??? note "Entra ID application management"
+
+    Using the Entra ID authentication protocol assumes that an Entra ID application is already set
+    up with the required permissions. Taipy doesn't manage the Entra ID application.
+
+    First, you need to
+    [create an application in the Microsoft Azure portal](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal)
+    within your organization.
+    Make sure that the Redirect URI of the application is set to `http://localhost`
```

Comment:
1- That is only relevant if we're dealing with a GUI application.
2- Authorized redirection URIs must be set to *<targetUrl>/* and *<targetUrl>/taipy-sso*
3- The app must have the *client_url* set to the deployment URL: use the *client_url* parameter to `Gui.run()` or run the application using `--client_url <url>`.